src/main/scala/com/microsoft/hyperspace/index/IndexSignatureProvider.scala

```diff
@@ -41,10 +42,10 @@ class IndexSignatureProvider extends LogicalPlanSignatureProvider {
    * @return signature, if both [[FileBasedSignatureProvider]] and [[PlanSignatureProvider]]
    *         can generate signature for the logical plan; Otherwise None.
    */
-  def signature(logicalPlan: LogicalPlan): Option[String] = {
+  def signature(logicalPlan: LogicalPlan): Option[Fingerprint] = {
```

Comment:
Ditto - Update the scaladoc.

Reply:
I think it doesn't need an update, since only the type is changed and the previous description can still apply.